tools/manifest/manifest.py

```diff
@@ -225,6 +225,9 @@ def load(tests_root, manifest):
                 rv = Manifest.from_json(tests_root, json.load(f))
         except IOError:
             return None
+        except ValueError:
```

Comment:
IMO, this should probably just capture `Exception`? @jgraham will probably disagree, so don't bother trying to be too prompt with addressing my comment, @fergald. 🙂

Reply:
I created another branch base on that suggestion.

https://github.com/fergald/web-platform-tests/tree/fergald-patch-2

Let me know what you think.